notebooks/Working with MONAI Auto3DSeg using MONAI Cloud API.ipynb

```diff
@@ -24,35 +24,121 @@
     "If you haven't already generated your key or if you're unsure about the process, follow our step-by-step guide for [Generating and Managing Your Credentials](./Generating%20and%20Managing%20Your%20Credentials.ipynb).\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "import json\n",
+    "import os\n",
     "import requests\n",
-    "\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Helper functions for running jobs\n",
+    "def wait_for_job(endpoint, headers, timeout):\n",
```

Comment:
Could we move the wait_for_job function closer to where we use it? It's confusing as the first set of code the user see.s